Runtime/Helpers/EnumHelper.cs

```diff
@@ -46,5 +46,22 @@ T max_inclusive
 
             return ( T )array.GetValue( UnityEngine.Random.Range( min_index, max_index + 1 ) );
         }
+
+        public static uint GetToggledFlagsCount<T>(
```

Comment:
if all flags are enabled, this will return 64 which seems to be an unexpected behaviour to me, should return the number of defined flags